code/testsuite/csheets/pf_Rogue.xml

```diff
@@ -925,7 +925,7 @@ BIO
 			<feats>+0</feats>
 			<magic_mod>+3</magic_mod>
 			<misc_mod>+1</misc_mod>
-			<race>+1</race>
+			<race>+0</race>
```

Comment:
When I see values changing, that still concerns me. Why are we losing a +1 to the racial saves?